README.md

```diff
@@ -0,0 +1,86 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard
+
+* The leaderboard website displays scores.
+
+# Screenshot
+
+![image](https://user-images.githubusercontent.com/74814780/136938704-30f68461-342b-44a4-a1a5-abadc4a6856d.png)
+
+
+## Setup
+
+### Clone this repository
+
+```bash
+$ git clone https://github.com/Ugboaja-Uchechi/leaderboard.git
+$ cd Leaderboard
+```
+
+### set up webpack
+
+```bash
+
+$ npm install --save-dev html-webpack-plugin  
+$ npm install --save-dev webpack-dev-server
+$ npm npm install --save-dev style-loader css-loader # To able to use npm run start for live reloading.
+```
```

Comment:
[OPTIONAL] Since `webpack` comes with its own live server node package, we don't need these packages anymore 👍🏽

Remove those steps before you merge to make your documentation even more professional. 😎